libbeat/docs/config-file-format.asciidoc

```diff
@@ -48,7 +48,7 @@ form is somewhat similar to JSON using `{}` for dictionaries and `[]` for lists:
 
 ["source","yaml",subs="attributes"]
 ------------------------------------------------------------------------------
-person: \{name: "John Doe", age: 34, country: "Canada"}
+person: {name: "John Doe", age: 34, country: "Canada"}
 colors: ["Red", "Green", "Blue"]
```

Comment:
@nik9000 This change works in asciidoctor, but throws errors in asciidoc. Not sure if there's some way to get the tagging to work for both.

The error in asciidoc is:

```
12:20:15 INFO:build_docs:asciidoc: WARNING: config-file-format.asciidoc: line 52: illegal system attribute name: name
```

Reply:
Does Asciidoc work if you drop `subs="attributes"`? It doesn't look like you use it.